chart/epinio/templates/_helpers.tpl

```diff
@@ -114,7 +114,7 @@ extraDeploy:
       labels:
         app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
         app.kubernetes.io/instance: {{ .Release.Name | quote }}
-        app.kubernetes.io/version: {{ default .Chart.AppVersion .Values.image.epinio.tag }}
```

Comment:
Should we remove this value then? https://github.com/epinio/helm-charts/blob/75b8fae6baf7734aa37012d8b1377e9d8a903313/chart/epinio/values.yaml#L11

I think we use it during development to point to the image built locally: https://github.com/epinio/epinio/blob/83b68f9510a3f1681eec0ef558a9f26188e14087/scripts/prepare-environment-k3d.sh#L84

Reply:
Ah, I guess i forgot to rebase! Missed it, thanks.

Reply:
Ah no, this is correct. This one was the only place where we don't need it.